README.md

```diff
@@ -43,46 +43,34 @@ The following libraries are optional:
 - (for legacy voice chat) speex - http://www.speex.org/downloads/
 
-Reducing the use of proprietary libraries is a focus of development and should
-be expected to change.
-
-All required libraries are available as precompiled binaries and
-associated files in the [development libraries bundle](https://github.com/H-uru/PlasmaPrefix/releases/download/2020.05.01/devlibs.zip)
-or can be built using their individual build instructions.
+All required libraries are available as [vcpkg](https://github.com/microsoft/vcpkg)
+ports or can be built using their individual build instructions.
 
 
 Compiling Instructions
 ----------------------
 
 Currently, compilation only targets Windows systems and requires Visual Studio
-2017 (including Visual Studio 2017 Community).
-
-**Quick-start instructions:**
-
-1. Run the `prepare_env.bat` script included in the repository.
-2. You should now have a *build* folder with a Visual Studio solution file
-   (.sln) inside.
-3. Open the solution in Visual Studio. You can compile CyanWorlds.com Engine by
-   pressing *Build -> Build Solution*. This will take some time.
-
-
-**To configure manually with CMake and build:**
-
-1. Start **CMake-GUI**.
-2. Set the *Where is the source code* option to the location where you cloned
-   the repository.
-3. Set the *Where to build the binaries* option to a subfolder of the
-   aforementioned location called *build*.
-4. Check the **Grouped** and **Advanced** options.
-5. Press **Configure**. Select *Visual Studio 15 2017* as the generator.
-6. Set the *CMAKE_INSTALL_PREFIX* option under CMAKE to the location where you
-   extracted the [development libraries bundle](https://github.com/H-uru/PlasmaPrefix/releases/download/2020.05.01/devlibs.zip).
-7. Press **Configure** again.
-8. Press **Generate**. You will now have a Visual Studio solution file (.sln)
-   in the folder that you specified to build the binaries in.
-9. Open the solution in Visual Studio. To build CyanWorlds.com Engine, find the
-   project labelled **INSTALL** in the *Solution Explorer*. Right click on this
-   project, and click **Build**.
+2019 (including Visual Studio 2019 Community).
+
+1. Clone the repository, including all submodules, in a git client or by
+   executing the following at the command line:
+   ```
+   git clone --recurse-submodules https://github.com/H-uru/Plasma.git
+   ```
+2. Open **Microsoft Visual Studio 2019**.
+3. Select **Open a local folder** and choose the folder where you cloned the
+   repository.
+4. Open the CMake Settings by chosing **Project > CMake Settings for Plasma**
+   from the Visual Studio menu bar.
+5. Add a new configuration in CMakeSettings.json by clicking the **green +**
+   button.
+6. Select **x86-Release** in the window that pops up.
+7. Compile the client by using **Build > Install Plasma** from the Visual
+   Studio menu bar.
+8. The client will be built and installed into the *out\install\x86-Release*
```

Comment:
Prefer to use backticks around the `out\install\x86-Release` path, because otherwise the backslashes look like vertical separators and it's hard to figure out what's intended.

(alternatively, use non-Windows slashes)

Reply:
Can you submit that as a suggestion, please?

Reply:
```suggestion
8. The client will be built and installed into the `out\install\x86-Release`
```